samples/CallingStateful/src/components/CallManager.tsx

```diff
@@ -67,7 +71,7 @@ const CallItem = (props: CallItemProps): JSX.Element => {
 
   useEffect(() => {
     if (call.state === 'Connecting') {
-      setCallTitle('Connecting');
+      setCallTitle('');
```

Comment:
Why this change?

Reply:
I don't have designs for this component so just building the experience that feels right, so this text having the call state where the person's name should be felt weird so made it a empty placeholder until the call connects